examples/pytorch/graphsaint/utils.py

```diff
@@ -0,0 +1,100 @@
+import json
+import os
+from functools import namedtuple
+import scipy.sparse
+from sklearn.preprocessing import StandardScaler
+import dgl
+import numpy as np
+import torch
+from sklearn.metrics import f1_score
+
+class Logger(object):
+    '''A custom logger to log stdout to a logging file.'''
+    def __init__(self, path):
+        """Initialize the logger.
+
+        Paramters
+        ---------
+        path : str
+            The file path to be stored in.
+        """
+        self.path = path
+
+    def write(self, s):
+        with open(self.path, 'a') as f:
+            f.write(str(s))
+        print(s)
+        return
+
+
+def save_log_dir(args):
+    log_dir = './log/{}/{}'.format(args.dataset, args.note)
+    os.makedirs(log_dir, exist_ok=True)
+    return log_dir
+
+
+def calc_f1(y_true, y_pred, multitask):
+    if multitask:
```

Comment:
For a sanity check, does multitask here always imply the prediction of multiple binary labels?